test/WebSites/Identity.DefaultUI.WebSite/PocoUserStartup.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
+using Microsoft.AspNetCore.Identity.InMemory;
+using Microsoft.AspNetCore.Identity.Test;
+using Microsoft.AspNetCore.Identity.UI;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Identity.DefaultUI.WebSite
+{
+    public class PocoUserStartup : StartupBase<PocoUser, IdentityDbContext>
+    {
+        public PocoUserStartup(IConfiguration configuration) : base(configuration)
+        {
+        }
+
+        public override void ConfigureServices(IServiceCollection services)
+        {
+            services.Configure<CookiePolicyOptions>(options =>
+            {
+                // This lambda determines whether user consent for non-essential cookies is needed for a given request.
+                options.CheckConsentNeeded = context => true;
+            });
+
+            services.AddDefaultIdentity<TestUser>()
```

Comment:
Isn't this supposed to be using PocoUser?

Reply:
TestUser = PocoUser, I saw there was already a store implementation for it on shared and reused it.

Reply:
I can rename the tests to TestUser but I think its better to convey all over the place that TestUser is a poco

Reply:
Just rename TestUser => PocoUser then

Reply:
Sounds good